feder/monitorings/models.py

```diff
@@ -31,6 +31,9 @@ def area(self, jst):
             case__institution__jst__lft__range=(jst.lft, jst.rght),
         )
 
+    def for_user(self, user):
```

Comment:
What's this for? I believe there is some reason for this method to exist, but the code does not suggest anything and there is no description, so I got bit confused here.

Reply:
Some time ago, Bartosz Wilk reported that he would like to have private monitoring (Unfortunatly, I can not find issue for that). I can see that private monitoring is a big thing that requires going through the entire application. I'm afraid that when we get down to it, we'll miss it and cause leak, so I introduced a method to go in that direction in small steps. In the future, we will introduce monitoring filtering based on some basic right, eg "view_monitoring".

That approach for permission via `QuerySet` was adopted in Watchdog Python project earlier ( https://github.com/search?l=Python&q=org%3Awatchdogpolska+for_user&type=Code ).

Reply:
Ok, I understand now. But a comment in the code would also be helpful there.